tests/common/api.hpp

```diff
@@ -121,7 +121,11 @@ class simple_matrix_base
       int_t jj = start2_ + j*stride2_;
       return data_[ii + jj*ld_];
     }
+#ifdef _MSC_VER
+protected:
```

Comment:
A simple comment here to explain why we need to change it to protected on Windows.